edgelet/provisioning/src/provisioning.rs

```diff
@@ -73,7 +73,7 @@ pub struct ManualProvisioning {
 
 impl ManualProvisioning {
     pub fn new(conn_string: &str) -> Result<Self, Error> {
-        ensure_not_empty!(conn_string);
+        ensure_not_empty!(conn_string, "Please provide a connection string");
```

Comment:
Total nit, but all of the other error messages have a `.` (period) at the end of the message. Does the error here do that?

Reply:
I would suggest helping the user by telling them exactly where they need to set the connection string.  Something like: "Please update your Edge .yaml file to include a connection string."

Reply:
@myagley , the text looks like this:
```
<3>2018-09-11T17:42:15Z [ERR!] - A provisioning error occurred.
<3>2018-09-11T17:42:15Z [ERR!] -        caused by: Utils error
<3>2018-09-11T17:42:15Z [ERR!] -        caused by: Argument is empty or only has whitespace - [Please provide a connection string]
```
So no - Complete sentences should end in periods - will fix.

@jongio Good idea, Thanks.